policy/policy.go

```diff
@@ -103,6 +103,14 @@ func (s Set) Get(p Policy) (string, bool) {
 	return v, ok
 }
 
+func (s Set) ToStringMap() map[string]string {
```

Comment:
Ah right, I see. Probably those types used as return types should be in `api`, but .. way bigger change. Let's put that on the refactor backlog.